### DISCUSSION TITLE
osu!lazer ignores OpenTabletDriver input

### DISCUSSION THREAD

**Discussion by obvMellow** [ppy/osu#29399] [Q&A] [answered]
I have OpenTabletDriver running however osu lazer ignores the tablet input. I can only get my tablet working when I enable tablet input from settings. I could simply use that but my pen is broken so it always registers the tip as being pressed. And since #14037 is not implemented yet I can't disable the tip binding. Enabling or disabling `High Precision Mouse` or changing `Screen mode` doesn't fix the issue.

I'm running Arch linux with Hyprland as the compositor

Here's a video demonstrating the issue:

https://github.com/user-attachments/assets/cd8bbe00-2ba1-4f66-886c-c30bd4c2f756

**Comment by Stoppedpuma** [comment, marked as answer]
Please try switching to "Artist Mode" in OpenTabletDriver.

**Reply by obvMellow** [reply]
That fixed it. Thanks!
